R/read-results.R

```diff
@@ -38,3 +38,25 @@ read_csv_test_results <- function(test_output_dir) {
   names(info) <- str_replace(basename(csv_files), fixed(".csv"), "")
   return(list(results = results, info = info))
 }
+
+#' Read manual test results.
+#' @param test_output_dir Directory that contains test subdirectories named by
+#'   test ID.
+#' @return Tibble with TestId and content columns. In the content, links to the
+#'   test's assets subdirectory are switched from relative to absolute paths.
+#' @importFrom purrr map_dfr
+#' @importFrom readr read_file
+#' @importFrom stringr str_replace_all fixed
+#' @export
+read_manual_test_results <- function(test_output_dir) {
+  testdirs <- list.files(test_output_dir, pattern = "^MAN-[A-Z]+-[0-9]+",
```

Comment:
need to trim any trailing `/` from the `test_output_dir` path for this work right. Otherwise, the image paths get an extra `/` in the middle. This seems to be `list.files()`'s fault.

```r
>   list.files("~/Downloads/metworx-manual-tests/", pattern = "^MAN-[A-Z]+-[0-9]+",
+                          full.names = TRUE)
[1] "/Users/SethG/Downloads/metworx-manual-tests//MAN-VSC-001"
[2] "/Users/SethG/Downloads/metworx-manual-tests//MAN-VSC-002"
[3] "/Users/SethG/Downloads/metworx-manual-tests//MAN-VSC-003"
[4] "/Users/SethG/Downloads/metworx-manual-tests//MAN-VSC-004"
[5] "/Users/SethG/Downloads/metworx-manual-tests//MAN-VSC-005"
[6] "/Users/SethG/Downloads/metworx-manual-tests//MAN-VSC-006"
>   list.files("~/Downloads/metworx-manual-tests", pattern = "^MAN-[A-Z]+-[0-9]+",
+                          full.names = TRUE)
[1] "/Users/SethG/Downloads/metworx-manual-tests/MAN-VSC-001"
[2] "/Users/SethG/Downloads/metworx-manual-tests/MAN-VSC-002"
[3] "/Users/SethG/Downloads/metworx-manual-tests/MAN-VSC-003"
[4] "/Users/SethG/Downloads/metworx-manual-tests/MAN-VSC-004"
[5] "/Users/SethG/Downloads/metworx-manual-tests/MAN-VSC-005"
[6] "/Users/SethG/Downloads/metworx-manual-tests/MAN-VSC-006"
```

weird behavior, but there you have it. You could consider using `fs::dir_ls()` instead, but then the regex behavior is slightly different. Or just manually check and trim. Up to you.

Ps- I don't think this matters for the `auto_test_dir` path

Reply:
> Otherwise, the image paths get an extra `/` in the middle

Huh, interesting.  Will update.  (Functionally I don't think this matters, given that `//+` is still valid and treated as `/`.)